content/get_start/UseCase_Workflows/pollination.md

```diff
@@ -0,0 +1,75 @@
+
+{{% notice info %}}
+Geometry Calculation setting is used to set the underlying engine that Pollination will use to parse the geometry. Full volume setting refers to Honeybee, while the Extruded Floor uses dragonfly.
+{{% /notice %}}
+
+
+**Extruded Floors** option extrudes the room/space/area boundaries with added apertures. 
+
+   - Apart from Extruded Floors option for geometry calculation, Detailed option for the window calculation method was selected. 
+   - The model was visualized to make sure there are no issues. To avoid having minor issues after the export such as gaps between floors and adjacency issues that can eventually cause the EnergyPlus simulation to fail, the model was previewed in plane and wireframe view as well and following steps were carried out. 
+      - The height was adjusted to close any gaps formed in the model due to the default heights. 
+      - The Align option was used to specify Reference Planes and Model Lines in Revit to pull the room/space boundaries to the nearest plane/line within a specified Align Distance defined in Pollination. This can help clean and fix the export without actually modifying the Revit model. 
+      - Solve Adjacency was set to Full. 
+
+![userdata](/BEM-for-PRM/get_start/UseCase_Workflows/images/DD_Extr_Step6_After_Fix.png?width=700px&align=right&classes=border,alignCenter)
+
+##### **2. Define the Construction Set, Program type, HVAC**
+Details related to "Construction Set", "Program Type" and "HVAC" were specified within Pollination.
+    
+  - [Construction Sets](https://docs.pollination.cloud/user-manual/rhino-plugin/energy-modeling/assign-construction-sets): Several options for construction templates with different vintages (based on ASHRAE 90.1 Baseline constructions in OpenStudio Standards) are available, and can also be edited. For this workflow, 2019 Vintage, Climate Zone 5 and Steel Framed type was selected. 
+
+  - [Program Types](https://docs.pollination.cloud/user-manual/rhino-plugin/energy-modeling/assign-program-types): Several options for load definitions and schedules for people, lighting, equipment, infiltration, ventilation, setpoint, service hot water corresponding to a list of space types and different vintages, are available and can be edited. Here, 2019 Vintage, with Medium Office as the Building Type and three program types - Conference, OpenOffice and Closed Office were selected accordingly for different spaces. 
+
+  - [HVAC](https://docs.pollination.cloud/user-manual/rhino-plugin/energy-modeling/assign-hvac-systems): Only "Ideal air loads" option is available by default and hence was selected (which was modeled as zone equipment). However, an HVAC system is a key element to run the PRM measure, thus, HVAC was revisited after exporting the model to OpenStudio. 
+    There is also an option in Pollination to import HVAC systems from OpenStudio that could be considered to assign HVAC systems. 
```

Comment:
Any link or instruction for this task?

Reply:
Are you referring to a link for "importing HVAC systems in Pollination"? If that's the case, the link attached for HVAC addresses that. That's how I got to know about it as well.

Reply:
You are right, Ammar. Good catch. 
For Revit Plugin, they have showed the same steps only in the video, so I will replace the links with this one then for all - https://docs.pollination.cloud/user-manual/revit-plugin/export-analytical-model

Reply:
Done!